### ISSUE TITLE
Will the code for evaluation be released?

### ISSUE THREAD

**Issue by tingxueronghua** [cientgu/InstructDiffusion#11]
I noticed that the script for reproducing segmentation results contains the seeds and lots of hyper-parameters. Will this take much influence when evaluating the model?

**Comment by Gengzigang** [comment]
Thank you for your attention to our work. For a single image, there is a relatively good seed. Given the large number of test-set images, the seed does not have a significant impact on the overall value. As for hyperparameters, you can use those provided in the script.
